fabric-connector/src/main/java/com/luxoft/fabric/model/ConfigData.java

```diff
@@ -125,14 +125,28 @@ public ChannelChaincode(String name, JsonNode collectionPolicy) {
 
     }
 
+    public static class ChannelPeer {
+
+        public final Map<String, Boolean> roles;
```

Comment:
Any guaranties on order? LinkedHashMap?

Reply:
Is there any reason why we should consider order here?